apps/core/tailwind.config.ts

```diff
@@ -152,6 +154,9 @@ export default {
 				3.75: '0.9375rem',
 				4.5: '1.125rem',
 				7.5: '1.875rem',
+				17: '4.25rem',
```

Comment:
where do 17,18, and 19 come from?

Reply:
17 * 4 = 68px
18 * 4 = 72px
19 * 4 = 76px
Tailwind values are divisible of 4